src/config/build_config.py

```diff
@@ -1,6 +1,6 @@
 # Please only set it to True if you are preparing an EI related PR
 # Do remember to revert it back to False before merging any PR (including EI dedicated PR)
-ENABLE_EI_MODE = False
+ENABLE_EI_MODE = True
```

Comment:
Will revert. Thanks for the reminder :)

Reply:
Oh, no, it's just a reminder for me, because GitHub PR UI "minimizes" a file once you mark it as reviewed. So it's easy to miss that this change to the `build_config.py` file was made and needs to be reverted on my final review.

Thanks!!